packages/scene-composer/src/interfaces/components.ts

```diff
@@ -1,3 +1,4 @@
+import { IIconLookup } from '../components/panels/scene-components/tag-style/IconPicker/IconPickerUtils/IconPickerUtils';
```

Comment:
better to just define `IIconLookup` in this file since it's the type for a data in tag scene component

Reply:
done